examples/zk.rs

```diff
@@ -25,6 +30,7 @@ arg_enum! {
     enum ProofAction {
         Prove,
         Verify,
+        Spartan,
```

Comment:
If the `arg_enum` macro allows it, I'd add a doc-comment saying that this action does both proving and verifying.

Reply:
Added. `arg_enum` allows one comment for the enum, but not individual ones for different options.